### PR TITLE
Last ad render time should be set only when the current ad finishes

### PR DESCRIPTION
**Pull request by hkaya**

### CODE REVIEW

File: test/index_spec.coffee [hunk 1]
```diff
@@ -142,6 +142,15 @@ describe 'AdView', ->
           expect(ad.creative_id).to.equal 'house'
           expect(ad.html5player.was_played).to.be.true
 
+        it 'should update the last render time', ->
+          @sandbox.stub @view, 'currentAd',
+            creative_id: 'house'
+          @sandbox.spy @view.proofOfPlay, 'write'
+
+          lrt = @view._lastAdRenderTime
```
Comment (pbogunovich) [inline test/index_spec.coffee]: I don't really have a problem with this, but doesn't moment JS have support for setting clock time for testing like this?

**Comment by pbogunovich** [comment]
+1

File: test/index_spec.coffee (hunk 1 above)
Comment (hkaya) [inline test/index_spec.coffee]: yea, sinon would be the proper way of mocking the clock. but I really don't care about exact times here.

**Comment by jawsthegame** [comment]
:+1: :+1:

**Comment by hkaya** [comment]
Thanks for the review guys!

**Pull request merged**